tests/php/src/DevTools/LikelyCulpritDetectorTest.php

```diff
@@ -71,12 +70,15 @@ public function test_analyze_backtrace() {
 
 		$previous_theme = get_stylesheet();
 		switch_theme( 'custom' );
+		require get_template_directory() . '/functions.php';
```

Comment:
I don't understand why this test every worked before without this `require`.

Reply:
Probably a false positive (due to caching/global state) that was "fixed" because of the change to `DependencyInjectedTestCase`.

Reply:
Hmm, then again, for this to actually happen, a frontend request would have to be rendered, wouldn't it?